streams/quickstart/java/pom.xml

```diff
@@ -26,7 +26,7 @@
     <parent>
         <groupId>org.apache.kafka</groupId>
         <artifactId>streams-quickstart</artifactId>
-        <version>2.3.2-SNAPSHOT</version>
+        <version>2.3.3-SNAPSHOT</version>
```

Comment:
Same as above.

Reply:
Same as above, it was already on 2.3.2, so I undid this change.